pkg/runtime/pubsub/bulksub.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/dapr/dapr/utils"
+
+	contribPubsub "github.com/dapr/components-contrib/pubsub"
+)
+
+const (
+	maxBulkCountKey     string = "maxBulkCount"
```

Comment:
Can we reuse keys that we have already defined in components-contrib? https://github.com/DeepanshuA/components-contrib/blob/5f50fd96aebdea11311a67ed4127f720398d15bf/pubsub/requests.go#L65